### PR TITLE
[HIBIKE] Recursively lint all Python files

### PR DESCRIPTION
**Pull request by baby-bell**
Just what the title says. This lints files in `hibike/` and all its subdirectories, as opposed to solely `hibike/`.

### CODE REVIEW

File: hibike/travis/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ artifacts-install:
 	$(nop)
 
 lint:
-	cd .. && pylint *.py
+	cd .. && find | grep "\\.py$$" | xargs pylint
```
Comment (kevinji) [inline hibike/travis/Makefile]: Does something like `pylint **/*.py` not work?

Comment (baby-bell) [reply hibike/travis/Makefile]: No, that only works for files one directory down. I would like to lint all Python files in the directory tree. There might be a way to remove the `grep`.

Comment (kevinji) [reply hibike/travis/Makefile]: Ah, I didn't realize that `**` only globs recursively by default on zsh; on bash, you'd need to set `globstar`. Does `find . -name "*.py" | xargs pylint` work?

Comment (baby-bell) [reply hibike/travis/Makefile]: Yes, that works. Thank you.

**Review by kevinji — APPROVED** [review]
lgtm

**Pull request merged**